cmd/ks-apiserver/app/server.go

```diff
@@ -98,6 +99,7 @@ func Run(s *options.ServerRunOptions) error {
 }
 
 func initializeAdminJenkins() {
+	devops.PreCheckJenkins()
```

Comment:
admin_jenkins.Client also check login

Reply:
Print more logs in the client method might be a better option

Reply:
this check is repeated, i remove it and go to `admin_jenkins.Client`.